cmd/talosctl/cmd/talos/diskusage.go

```diff
@@ -41,7 +41,7 @@ var duCmd = &cobra.Command{
 			}
 
 			stream, err := c.DiskUsage(ctx, &machineapi.DiskUsageRequest{
-				RecursionDepth: recursionDepth,
+				RecursionDepth: recursionDepth + 1,
```

Comment:
this will cause some fun inconsistencies if `talosctl` and Talos versions dont' match, but I think it's better to fix things now rather than keep them broken forever.